NR1 sweep | DO NOT MERGE (#7524)
newrelic/docs-website: src/content/docs/browser/new-relic-browser/browser-agent-spa-api/add-page-action.mdx

      </td>

      <td>

>         Optional. JSON object with one or more key/value pairs. For example: `{key:"value"}`. The key will be reported as its own PageAction attribute with the specified values.

Reviewer: ```suggestion
        Optional. JSON object with one or more key/value pairs. For example: `{key:"value"}`. The key is reported as its own PageAction attribute with the specified values.
```
